miio/airhumidifier.py

```diff
@@ -199,8 +199,8 @@ def water_level(self) -> Optional[int]:
 
         If water tank is full, depth is 125.
         """
-        if self.depth is not None and self.depth <= 125:
-            return int(self.depth / 1.25)
+        if self.data.get("depth") is not None and self.data["depth"] <= 125:
```

Comment:
Introduce a new local variable `depth` to avoid typing `self.data` again:
```
depth = self.data.get("depth")
...
```

Otherwise looks ok for me :-)